src/main/java/tools/MaplePacketCreator.java

```diff
@@ -3661,31 +3667,6 @@ private static void writeLongMaskChair(final MaplePacketLittleEndianWriter mplew
                 return pOutPacket.getPacket();
         }
 
-        public static byte[] getWhisper(String sender, int channel, String text) {
```

Comment:
Can't compile due to this renaming to `getWhisperReceive`. Issue is in method `net.server.world.World#whisper` which uses the old method.